StreamChatUITestsAppUITests/Tests/MessageList_Tests.swift

```diff
@@ -501,6 +501,26 @@ extension MessageList_Tests {
         }
     }
     
+    func test_paginationOnThread() throws {
+        linkToScenario(withId: 55)
+        
+        try XCTSkipIf(ProcessInfo().operatingSystemVersion.majorVersion == 12,
+                      "[CIS-2020] Scroll on message list does not work well enough")
+        
+        let replyCount = 60
+        
+        GIVEN("user opens the channel") {
+            backendRobot.generateChannels(count: 1, messagesCount: 1, replyCount: replyCount)
```

Comment:
My only concern with this function is that it is not very flexible, you can't decide which messages have replies, and which don't. For these use case, it is fine now, but if we want to add more complex scenarios it will start to be difficult to use this function 🤔 Maybe creating a tech debt ticket would be nice for this

Reply:
Yeah, it is a bit tricky func, makes sense to create a separate ticket for improving it.